src/OrchardCore.Commerce.Web/OrchardCore.Commerce.Web.csproj

```diff
@@ -18,7 +18,7 @@
   </ItemGroup> -->
 
   <ItemGroup>
-    <PackageReference Include="Lombiq.NodeJs.Extensions" Version="1.0.0-alpha.occ-15.3" />
+    <PackageReference Include="Lombiq.NodeJs.Extensions" Version="1.0.0" />
```

Comment:
Please also remove the following property assignments above:

```xml
<NodeJsExtensionsMarkdownAnalysisMode>solution</NodeJsExtensionsMarkdownAnalysisMode>
<ExecDotnetPostcleanCommand>false</ExecDotnetPostcleanCommand>
<ExecDotnetPrebuildCommand>false</ExecDotnetPrebuildCommand>
<NodeJsExtensionsGlobalESLintConfigurationDirectory>false</NodeJsExtensionsGlobalESLintConfigurationDirectory>
<NodeJsExtensionsCreateStylelintConfigurationFile>false</NodeJsExtensionsCreateStylelintConfigurationFile>
```

Reply:
Thank you!